float64 shoulder_pan_joint
float64 shoulder_lift_joint
float64 upperarm_roll_joint
float64 elbow_flex_joint
float64 forearm_roll_joint
float64 wrist_flex_joint
float64 wrist_roll_joint
---